test/bundler/esbuild/default.test.ts

```diff
@@ -2786,6 +2786,37 @@ describe("bundler", () => {
       bunArgs: ["--define", 'import.meta.url="url_here"', "--define", 'import.meta.path="path_here"'],
     },
   });
+  itBundled("default/ImportMetaEnv", {
```

Comment:
can you add a second test that uses `target: 'bun'`. i believe by default that target will inline all environment variables. i'm not sure what should be followed with import.meta.env, but i would like a test for it.